libraries/door43_tools/project_deployer.py

```diff
@@ -126,7 +126,7 @@ def deploy_revision_to_door43(self, build_log_key):
                     content += '<h1 class="conversion-requested">{0}</h1>'.format(build_log['message'])
                     content += '<p><i>No content is available to show for {0} yet.</i></p>'.format(repo_name)
                     content += """
-                    <script type="text/javascript">setTimeout(function(){window.location.reload(1);}, 10000);</script>
+                    <script type="text/javascript">CONV_REQUESTED=setTimeout(function(){window.location.reload(1);}, 10000);</script>
```

Comment:
Don't get what you're doing here. We should do away with any 10 second reloading, thus just completely delete this line (and the line before and after) and just have a loop in the project_page_functions.js be triggered if the status is "started" or "requested" (maybe go by success? Can't remember if it is null if not completed) until either 10 minutes has elapsed since started_at or the status changes.  I think you're in agreement with that, so surprised to see this still here. Reloading the page increases page view count.